schema/base/land_cover.yaml

```diff
@@ -0,0 +1,50 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: Land Cover Schema
+description: representation of the Earth's natural surfaces 
+type: object
+properties:
+  id:
+    "$ref": ../defs.yaml#/$defs/propertyDefinitions/id
+  geometry:
+    description: classification of surface, can correspond to the 'natural' tag from OpenStreetMap
+    unevaluatedProperties: false
+    oneOf:
+      - "$ref": https://geojson.org/schema/Polygon.json
+      - "$ref": https://geojson.org/schema/MultiPolygon.json
+  properties:
+    unevaluatedProperties: false
+    allOf:
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/overtureFeaturePropertiesContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/levelContainer
+    required:
+      - subType
+    properties:
+      subType:
+        description: type of surface represented
+        type: string
+        enum:
+          - barren
+          - crop
+          - forest
+          - grass
+          - mangrove
+          - moss
+          - shrub
+          - snow
+          - urban
+          - wetland
+      cartography:
```

Comment:
Comments/ideas:

1. [ ] Isn't this invalid/ignored JSON schema? I think `cartography` requires below it a `type: object` clause, and a `properties: ...` clause under which the properties should be listed?
2. [ ] Should `cartography` be `required` for land_cover?
3. [ ] Should `cartography` have a description? Along the lines of "Contains cartographic hints for optimum use of the feature within a map"...
4. [ ] A possible reason why #1 and #2 are causing issues is because of a lack of examples/counterexamples for `land_cover`. Can you add some?